py3status/modules/xscreensaver.py

```diff
@@ -11,45 +11,44 @@
 Settings can be managed using "xscreensaver-demo".
 
 Configuration parameters:
-    cache_timeout: refresh interval for this module (default 60)
     format: default format for this module (default '{output}')
-    string_off: display when Xscreensaver is not running (defaults '🖥️')
-    string_on: display when Xscreensaver is running (defaults '🖥️')
+    format_off: display when Xscreensaver is not running (default '')
+    format_on: display when Xscreensaver is running (default '')
 
  Format placeholders:
     {output} display xscreensaver output
 
 @author neutronst4r <c7420{at}gmx{dot}net>
 """
 from os import setpgrp
-from subprocess import Popen, DEVNULL, check_output, CalledProcessError
+from subprocess import Popen, DEVNULL
 
 
 class Py3status:
     """
     """
     # available configuration parameters
-    cache_timeout = 60
```

Comment:
Does this mean dpms should be fixed too? It's missing a `cache_timeout` too.
That's where I got the clue from.

I was trying to avoid this https://github.com/ultrabug/py3status/issues/580 by optimizing a bit. Getting it to not run something every ___ seconds, but to be updated only on click.

I think if somebody is planning on using this xscreensaver module, then they would be only xscreensaver to stop or run xscreensaver... basically CACHE_FOREVER.

It'll be back.